packages/manager/modules/billing/src/billing/autoRenew/agreements/details/details.module.js

```diff
@@ -0,0 +1,11 @@
+import routing from './details.routing';
+import controller from './user-agreements-details.controller';
+
+const moduleName = 'ovhManagerBillingAgreementsDetails';
+
+angular
+  .module(moduleName, ['ui.router'])
+  .config(routing)
+  .controller('UserAccount.controllers.agreements.details', controller);
```

Comment:
Could the name be uniformized ? something like 
```suggestion
  .controller('UserAccountAgreementsDetailsController', controller);
```